cli/azd/.gitignore

```diff
@@ -1,3 +1,5 @@
 azd
 azd.exe
-build
\ No newline at end of file
+build
```

Comment:
Tip: keep a terminating new line to make future diffs (and merges) cleaner. Personally, I like to always keep a _.editorconfig_ in my root with some settings that help enforce that.